nimare/meta/kernel.py

```diff
@@ -499,3 +499,98 @@ def _generate_description(self):
             "For voxels with overlapping spheres, the maximum value was retained."
         )
         return description
+
+
+class SDMKernel(KernelTransformer):
+    """Generate SDM modeled activation images from coordinates.
+
+    This kernel generates effect size maps using an anisotropic Gaussian kernel
+    as described in the SDM literature. By default, uses a FWHM of 20mm following
+    the SDM-PSI implementation.
+
+    .. versionadded:: 0.3.1
+
+    Parameters
+    ----------
+    fwhm : :obj:`float`, optional
+        Full-width half-max for Gaussian kernel in mm. Default is 20mm as used
+        in SDM-PSI.
+    memory : instance of :class:`joblib.Memory`, :obj:`str`, or :class:`pathlib.Path`
+        Used to cache the output of a function. By default, no caching is done.
+        If a :obj:`str` is given, it is the path to the caching directory.
+    memory_level : :obj:`int`, default=0
+        Rough estimator of the amount of memory used by caching.
+        Higher value means more memory for caching. Zero means no caching.
+
+    References
+    ----------
+    .. footbibliography::
+
+    Notes
+    -----
+    This is a simplified implementation of the SDM kernel. The full SDM-PSI algorithm
+    includes additional steps such as multiple imputation and subject-level simulation
+    which are not implemented here.
```

Comment:
Is there more that needs to be implemented in the kernel? or is this note outdated?